specification/modsimworkbench/resource-manager/Microsoft.ModSimWorkbench/preview/2021-03-01-preview/fileRequests.json

```diff
@@ -0,0 +1,493 @@
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "workbenchName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the workbench."
+          },
+          {
+            "name": "chamberName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the chamber."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request is successful. The operation returns the list of all files",
+            "schema": {
+              "$ref": "#/definitions/RequestPageableList"
+            }
+          },
+          "default": {
+            "description": "Default Error Response",
+            "schema": {
+              "$ref": "./modsimworkbenches.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List all files": {
+            "$ref": "./examples/FileRequestsList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RequestPageableList": {
+      "description": "Microsoft.Sdw.Core.ResourceMetadata.PageableList implementation specific to pageable lists of Microsoft.Sdw.Core.ResourceMetadata.AzureResource`1.",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "Gets or sets the URL (link) to the next result set.",
+          "type": "string"
+        },
+        "value": {
+          "description": "Gets or sets the values to emit from this list.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Request"
+          },
+          "x-ms-identifiers": [
+            "value"
+          ]
+        }
+      }
+    },
+    "FileRequestsManage": {
+      "description": "Manage download file requests in an chamber Data pipeline.",
+      "type": "object",
+      "properties": {
+        "result": {
+          "type": "string",
+          "description": "Manage download file requests in an chamber Data pipeline action"
+        }
+      }
+    },
+    "FileDownloadSasUri": {
+      "description": "Get SAS URI to download a file from an chamber data pipeline.",
+      "type": "object",
+      "properties": {
+        "result": {
+          "type": "string",
+          "description": "Get SAS URI to download a file action"
```

Comment:
SAS uris should be marked as x-ms-secret right?

Reply:
sasUriId value is not SAS URI, it's key vault secret ID for the SAS URI: not secretssasUriId is internalMetadata and should not be in Swagger - will remove